src/Commands/ExportPostmanCommand.php

```diff
@@ -161,7 +174,10 @@ public function handle(): void
             }
         }
 
-        Storage::disk($this->config['disk'])->put($exportName = "postman/$this->filename", json_encode($this->structure));
+        Storage::disk($this->config['disk'])->put(
+            $exportName = "postman/$this->filename",
+            json_encode($this->structure)
+        );
```

Comment:
What was the point of reformatting this?

Reply:
Reverting, my IDE got a bit excited with the long line. :)